module.js

```diff
@@ -10,53 +10,70 @@ var parseSource = require('./sourceParse');
 var token = 'token ' + process.env.OAUTH_TOKEN;
 var productName = 'Discord';
 
-var hook = function(request, response) {
+var hook = function(httpRequest, response) {
     var localToken = token;
     response.status(200).send('ok');
-    github(request.body, localToken);
+    github(httpRequest, localToken);
 };
 
-var github = function(payload, localToken) {
-    var headCommitURL = payload.repository.commits_url.replace('{/sha}', '/' + payload.head_commit.id);
+var github = function(httpRequest, localToken) {
+    var eventType = httpRequest.headers['x-github-event'];
+    var payload = httpRequest.body;
+    var configMetadataURL = payload.repository.contents_url.replace('{+path}', '.doiuse');
+    var commitsURL, commentURL;
 
-    console.log("Repo: " + payload.repository.full_name);
+    console.log('Repo: ' + payload.repository.full_name);
 
     request({
-        url: headCommitURL,
+        url: configMetadataURL,
         headers: {
             'User-Agent': productName
         }
     }, function(error, response, body) {
-        var parsedBody = JSON.parse(body);
-        var files = parsedBody.files;
-        var configMetadataURL = payload.repository.contents_url.replace('{+path}', '.doiuse');
-        request({
-            url: configMetadataURL,
-            headers: {
-                'User-Agent': productName
-            }
-        }, function(error, response, body) {
-            var configMetadata = JSON.parse(body);
-            var config = ['last 2 versions'];
+        var configMetadata = JSON.parse(body);
+        var config = ['last 2 versions'];
 
-            if (configMetadata.content) {
-                var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
-                config = configMetadataContent.replace(/\r?\n|\r/g, '').split(/,\s*/);
-            }
+        if (configMetadata.content) {
+            var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
+            config = configMetadataContent.replace(/\r?\n|\r/g, '').split(/,\s*/);
```

Comment:
I'd probably add some sort of comment about why you're doing this whitespace stuff.